dockerfiles/Dockerfile-toolchain

```diff
@@ -95,6 +95,7 @@ RUN eval $(opam env)
 
 # Install OPAM packages for spacetime compiler; use ulimited stack for camlp4 build
 ADD /src/opam.export .
+RUN opam install dune.1.11.4
```

Comment:
dune is installed separately to allow the import to work?

Reply:
Yes -- the opam import doesn't work without this :/